src/towncrier/build.py

```diff
@@ -279,6 +279,9 @@ def should_remove_fragment_files(
     answer_yes: bool,
     answer_keep: bool,
 ) -> bool:
+    if not fragment_filenames:
+        click.echo("No news fragments to remove. Skipping...")
```

Comment:
Heh, this was my first take too and then I told you the other way because it seemed simpler. This is fine though.
If I was nit-picking, I wouldn't finish with '...' as that is used everywhere else in the process to indicate something is about to be done.

Reply:
Fair enough! Replaced `...` with `!` and force-pushed.